indra/llfilesystem/llfilesystem.cpp

```diff
@@ -325,3 +320,64 @@ bool LLFileSystem::remove()
 
     return true;
 }
+
+void LLFileSystem::updateFileAccessTime(const std::string file_path)
```

Comment:
Forgot to also change it to a const ref or even string_view?

Reply:
Good catch! Fixed!